### ISSUE TITLE
Extend to the Humble Store

### ISSUE THREAD

**Issue by forteller** [nathanielw/humble-enhancer#6]
Thank you for this fantastically time saving script! If it would be possible for you to extend it to work on each individual games page in the Humble Store too that would be just amazing!

**Comment by nathanielw** [comment]
Yes! This is something I had thought about/planned when I started building this. Not much time at the moment to work on it, but it's definitely something I want to add.

**Comment by johnsaucier** [comment]
I would love to see this added, it would help when I'm going through all the games to know which I have already.
